benchmarking/commons/hpo_main_local.py

```diff
@@ -38,6 +38,8 @@ def get_benchmark(
         benchmark_kwargs["n_workers"] = args.n_workers
     if args.max_wallclock_time is not None:
         benchmark_kwargs["max_wallclock_time"] = args.max_wallclock_time
+    if args.instance_type is not None:
+        benchmark_kwargs["instance_type"] = args.instance_type
     return benchmark_definitions(**benchmark_kwargs)[args.benchmark]
 
```

Comment:
OK

Reply:
I think in general we need to find a good middle ground (stating what has changed is good).
Because while I agree that PRs take a bit longer when some smaller things are also done, it is also the case (in this team at least) that in general, PRs are lying around quite some time, so we cannot afford to open many tiny ones either, this would just lead to delays.

Reply:
Makes sense, this is why I suggested to mention any changes beyond the primary role of the PR into the description to make sure its clear why they are there.